clustermesh/certs.go

```diff
@@ -183,25 +183,21 @@ func (k *K8sClusterMesh) deleteCertificates(ctx context.Context) error {
 }
 
 func (k *K8sClusterMesh) installCertificates(ctx context.Context) error {
-	err := k.certManager.LoadCAFromK8s(ctx)
+
+	caSecret, err := k.certManager.GetCASecret(ctx)
 	if err != nil {
-		if !k.params.CreateCA {
-			k.Log("❌ Cilium CA not found: %s", err)
-			return err
-		}
-
-		k.Log("🔑 Generating CA...")
-		if err := k.certManager.GenerateCA(); err != nil {
-			return fmt.Errorf("unable to generate CA: %w", err)
-		}
-
-		if err := k.certManager.StoreCAInK8s(ctx); err != nil {
-			return fmt.Errorf("unable to store CA in secret: %w", err)
-		}
-	} else {
-		k.Log("🔑 Found existing CA in secret %s", defaults.CASecretName)
+		k.Log("❌ Unable to get Cilium CA: %s", err)
```

Comment:
I think this log message might be misleading. It's not the CA it's the CA secret, right?

Reply:
I'm all for being more verbose. I've updated this log message